docs/site/Server.md

```diff
@@ -290,21 +290,22 @@ for more details.
-| protocol          | string (http/https)       | Specify the protocol on which the RestServer listens for traffic.                                         |
-| basePath          | string                    | Specify the base path that RestServer exposes http endpoints.                                             |
-| key               | string                    | Specify the SSL private key for https.                                                                    |
-| cert              | string                    | Specify the SSL certificate for https.                                                                    |
-| cors              | CorsOptions               | Specify the CORS options.                                                                                 |
-| sequence          | SequenceHandler           | Use a custom SequenceHandler to change the behavior of the RestServer for the request-response lifecycle. |
-| openApiSpec       | OpenApiSpecOptions        | Customize how OpenAPI spec is served                                                                      |
-| apiExplorer       | ApiExplorerOptions        | Customize how API explorer is served                                                                      |
-| requestBodyParser | RequestBodyParserOptions  | Customize how request body is parsed                                                                      |
-| router            | RouterOptions             | Customize how trailing slashes are used for routing                                                       |
-| listenOnStart     | boolean (default to true) | Control if the server should listen on http/https when it's started                                       |
+| Property            | Type                      | Purpose                                                                                                                                |
+| ------------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| host                | string                    | Specify the hostname or ip address on which the RestServer will listen for traffic.                                                    |
+| port                | number                    | Specify the port on which the RestServer listens for traffic.                                                                          |
+| protocol            | string (http/https)       | Specify the protocol on which the RestServer listens for traffic.                                                                      |
+| gracePeriodForClose | number                    | Specify the grace period in milliseconds to allow the RestServer to finish processing in-flight requests before closing the connection |
```

Comment:
Please capture this information from code comments too:

> Defaults to Infinity (don't force-close). If you want to immediately kill all sockets you can use a grace of 0.

and 
> See https://www.npmjs.com/package/stoppable

I am also not sure if the current description conveys the important information that `gracePeriodForClose` applies only to server shutdown?